descriptive-analysis.qmd

```diff
@@ -6,19 +6,19 @@ editor: visual
 
 ## Introduction
 
-This document presents the descriptive results from the analysis of 4 different tools (`sciscore`, `trialidentifier`, `ctregistries` and `nct`). The aim of these tools is to screen papers for one or more registration IDs (protocol, trial registry, etc).
+This document presents the descriptive results from the analysis of 4 different tools (`sciscore`, `TRNscreener`, `ctregistries` and `nct`). The aim of these tools is to screen papers for one or more registration IDs (protocol, trial registry, etc).
 
 #### Data description
 
-This analysis was limited to papers (n = 117) where a hit was found by one or more tools. Overall, 200 IDs were analyzed. These IDs were grouped in a data file named `regset.csv`, with the following variables/columns:
+This analysis was limited to papers (n = 117) where a hit was found by one or more tools. Overall, 199 IDs were analyzed. These IDs were grouped in a data file named `regset.csv`, with the following variables/columns:
```

Comment:
Added the text. However, what exactly do you mean by "_unique per paper_"? 
I want to clarify it in a note so that the reader knows what it is.

Reply:
some IDs were found across >1 paper, so we need to make it clear to the reader that we mean 199 distinct ID-paper combinations. The alternative would be to say that 196 unique IDs were analysed, but since the paper level is relevant for the analysis (since we also look into sections of the paper were detected), I lean towards sticking with the phrasing above. Happy to consider other ways to put this across! Does that clarify?